### PR TITLE
New instruction file for Nekbone

### PR DESCRIPTION
**Pull request by tenbrugg**
Signed-off-by: Monika ten Bruggencate tenbrugg@gmail.com

### CODE REVIEW

**Comment by tenbrugg** [comment]
@sungeunchoi 
relates to #15

File: scripts/miniapps/NOTESnekbone.txt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+   > srun -n 216 --exclusive <path/exec> ex1
+
+   b) Suitable for KNL systems
+   > srun -n 2048 --ntasks-per-node=64 --hint=nomultithread --exclusive <path/exec> ex1
+
+   > srun -n 8192 --hint=nomultithread --exclusive <path/exec> ex1
+
+8. Before terminating, the app prints
+   " Exitting...."
```
Comment (sungeunchoi) [inline scripts/miniapps/NOTESnekbone.txt:72]: Does it really mis-spell "exitting"?

**Comment by tenbrugg** [comment]
Yes, it really does mis-spell exiting and I copied the output literally

**Comment by sungeunchoi** [comment]
Huh.  Maybe it's some alternate conjugation that we're not familiar with.

👍

**Pull request merged**